handlers/product-move-api/ProductMovementAPI.yaml

```
@@ -0,0 +1,134 @@
+openapi: 3.0.3
+
+info:
+  title: Product Movement API
+  description:
+    API to facilitate replacing an existing subscription 
+    with another subscription to a different type of product.
+  version: 0.0.1
+
+paths:
+
+  /eligibles/{subName}:
```

doesn't quite seem clear what it does from the "eligibles", perhaps we can be a little more verbose as it won't be too costly.
```suggestion
  /availableSwitches/{subName}:
```
or `listSwitches`?
or other suggestions

---

after a bit of [googleing](https://www.google.com/search?q=rest+best+practices) I wonder if we should have a list of possible switches
```suggestion
  /{subscriptionName}/switches:
```
and then a POST or PUT to the same endpoint to do a switch?

I'm not sure how well we fit in with the Restful principles, they seem to be more database oriented, but worth thinking about.

---

I think `availableProductMoves` would be better as makes it consistently a `move` rather than a `switch`.
Then perhaps `productMove` as the actual move endpoint.

---

Done this.